lib/ansible/galaxy/collection/__init__.py

```diff
@@ -1405,26 +1457,32 @@ def _build_collection_dir(b_collection_path, b_collection_output, collection_man
     return collection_output
 
 
-def _normalize_collection_path(path):
+def _normalize_collection_path(path: pathlib.Path | str) -> pathlib.Path:
     str_path = path.as_posix() if isinstance(path, pathlib.Path) else path
     return pathlib.Path(
         # This is annoying, but GalaxyCLI._resolve_path did it
         os.path.expandvars(str_path)
     ).expanduser().absolute()
 
 
-def find_existing_collections(path_filter, artifacts_manager, namespace_filter=None, collection_filter=None, dedupe=True):
+def find_existing_collections(
+    path_filter: str | t.Sequence[str],
```

Comment:
ansible-doc can call it with None
```suggestion
    path_filter: str | t.Sequence[str] | None,
```

Reply:
Thanks! Do you mean this place? https://github.com/ansible/ansible/blob/82b64d4b69deffb136417947ed7203b9bfe42fcf/lib/ansible/collections/list.py#L60

I can make this change. Otherwise, it'd pop up when that portion of the codebase is being typed..